h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -1,29 +1,77 @@
+import { useId } from 'preact/hooks';
+
 import { Button, Input, Textarea } from '@hypothesis/frontend-shared';
 
 export default function CreateGroupForm() {
+  const nameId = useId();
+  const descriptionId = useId();
+
   return (
-    <>
-      <h1>Create a new private group</h1>
+    <div className="text-grey-6 text-sm/tight">
+      <h1 className="mt-14 mb-8 text-grey-7 text-xl/none">
+        Create a new private group
+      </h1>
 
       <form>
-        <div>
-          <label for="name">Name*</label>
-          <Input id="name" autofocus autocomplete="off" required />
-          0/25
+        <div className="mb-4">
```

Comment:
I think in future we're going to want to tweak the label or the field container so there is a bit more spacing between the bottom of the label and the top of the input field. I suggest to leave as-is in this PR and we can tweak it afterwards. This might involve creating a reusable "form field container" component to handle this.

Reply:
We may want to replace examples like "Input with label" and "Textarea with label" in the component library.

I'd currently find it hard to decide how much spacing to put between a label and its form field, without any vertical rhythm or grid (that I'm aware of) it just seems like an arbitrary choice. Perhaps a defined grid that all components stick to is something we're missing?